backend/app/models/anti_recommendation.py

```diff
@@ -1,7 +1,7 @@
-from pydantic import BaseModel
+from app.models.record import Record
 
 
-class AntiRecommendation(BaseModel):
+class AntiRecommendation(Record):
     """Pydantic Model to hold an anti-recommendation."""
 
     title: str = ""
```

Comment:
Rather than have a sentinel here, just skip the return.

Reply:
I have corrected the structure of `AntiRecommendation`.